root/static/scripts/tests/Control/URLCleanup.js

```diff
@@ -1006,6 +1006,28 @@ const testData = [
     expected_relationship_type: 'license',
             expected_clean_url: 'https://creativecommons.org/publicdomain/zero/1.0/',
   },
+  // DAHR
+  {
+                     input_url: 'https://adp.library.ucsb.edu/index.php/talent/detail/800/Louis_Armstrong_All-Stars_Musical_group',
+             input_entity_type: 'artist',
+    expected_relationship_type: 'otherdatabases',
+            expected_clean_url: 'https://adp.library.ucsb.edu/index.php/talent/detail/800',
+       only_valid_entity_types: ['artist'],
+  },
+  {
+                     input_url: 'https://adp.library.ucsb.edu/index.php/matrix/refer/2000308570',
```

Comment:
Please test with a hash as it is used internally to collapse lists:

```suggestion
                     input_url: 'https://adp.library.ucsb.edu/index.php/matrix/refer/2000308570#',
```